configs/sites/emc-rhel/modules.yaml

```diff
@@ -0,0 +1,15 @@
+modules:
+  default:
+    enable::
+    - tcl
```

Comment:
Which one are you using, tcl or lmod? I guess only one of them. I'd throw out the other one together with the corresponding section below.

Reply:
The default is tcl, but at least some users prefer lmod... I'll see if the tcl modules all work with lmod anyway, in which case I'll throw out lmod.

Reply:
Noooo. Throw out tcl. Please.

Reply:
Based on my testing it looks like the tcl modules are all compatible with lmod (all the logic is pretty basic), but the reverse is not true. @aerorahul unless you have a specific need for the lua format (like adding certain functionality to the modules), then I'd be inclined to support both tcl and lmod users by using tcl format. I'm not necessarily averse to having it only build the lmod modules, but it makes it a little more likely that I'll get confused questions from users who use the system default...

Reply:
If your rhel has lmod support, then use lmod please.